Code/C++/binary_tree_BFS_traversal.cpp

```diff
@@ -0,0 +1,112 @@
+#include <bits/stdc++.h>
```

Comment:
add a generic description at the top. See other code samples in the repo for reference

Reply:
done, please review